javatests/arcs/core/util/MutableBiMapTest.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.core.util
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+/** Tests for [MutableBiMap]. */
+@RunWith(JUnit4::class)
+class MutableBiMapTest {
+
+    @Test
+    fun mutableBiMap_supportsStandardMapOps() {
```

Comment:
In general it's good to split things into individual tests of specific functionality, rather than one big test method for all scenarios.

Reply:
Done :)